src/components/20-molecules/datepicker/index.js

```diff
@@ -729,6 +752,55 @@ class AXADatepicker extends NoShadowDOM {
     // any preexisting error should be cleared
     this.error = null;
   }
+
+  handleNavigateToPrevMonth() {
+    this.navigateMonth(-1);
+  }
+
+  handleNavigateToNextMonth() {
+    this.navigateMonth();
+  }
+
+  // Navigates to the next allowed month, when offset = 1.
+  // Navigates to the previous allowed month, when offset = -1.
+  // Has no effect, when there is no next/previous allowed month.
+  navigateMonth(offset = 1) {
+    const MIN = 0;
+    const MAX = 11;
+    const { min, max, abs } = Math;
+
+    let month = parseInt(this.month, 10) + offset;
+    let year = parseInt(this.year, 10);
+
```

Comment:
Here starts a block of very dense logic. You need to ease readability with a series of comments that convey the *intent* of your code. I make some suggestions in what follows:

// month out of bounds ?